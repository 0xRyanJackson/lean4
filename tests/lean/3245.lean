example (b1 b2 : Bool) :
  (if b1 then 0 else (if b2 then 0 else if b2 then 42 else 0)) = 0 := by
  split
  · rfl
  trace_state
  · split <;> rfl
