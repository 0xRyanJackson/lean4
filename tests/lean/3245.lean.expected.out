case inr
b1 b2 : Bool
h✝ : ¬b1 = true
⊢ (if b2 = true then 0 else if b2 = true then 42 else 0) = 0
